.gitignore

```diff
@@ -1,8 +1,6 @@
 node_modules
-client/index.html
 .tmp
 .idea
-client/app/app.less
```

Comment:
We can also remove `server/*` occurrences.

Reply:
done